### PR TITLE
PIM-7082: remove double user menu on product import edit form

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

Fix double user menu on product import
Fix access to product edit form when default locale is not viewable
Fix on attribute removal and re-creation with another type

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php [hunk 1]
```diff
@@ -97,6 +98,13 @@ public function createFromStorageFormat(array $rawValues)
                                     $e->getPropertyValue()
                                 )
                             );
+                        } catch (InvalidPropertyTypeException $e) {
```
Comment (jjanvier) [inline src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php]: missing spec... at least

Comment (juliensnz) [reply src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php]: yep yep, told it on slack, I still need to fix add specs and behats ;)

File: src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php [hunk 2]
```diff
@@ -97,6 +98,13 @@ public function createFromStorageFormat(array $rawValues)
...
+                            $this->logger->warning(
+                                sprintf(
+                                    'Tried to load a product value with the property "" that does not have the good type.'
```
Comment (grena) [inline src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php]: It misses something between the `""` no?

File: src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php (hunk 1 above)
Comment (juliensnz) [reply src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php]: done

Comment (jjanvier) [reply src/Pim/Component/Catalog/Factory/ValueCollectionFactory.php]: a PR is not to review is there is no test...

**Pull request merged**